src/solver/types.jl

```diff
@@ -561,6 +564,7 @@ Base.@kwdef struct AtmosModel{
     sfc_temperature::ST = nothing
     insolation::IN = nothing
     surface_model::SM = nothing
+    surface_flux_model::SFM = nothing
```

Comment:
I would prefer we didn't introduce yet another "surface" model setting. It adds complexity and confusion and makes coupling harder.

All the configurations related to the surface should be in a single place (this should include `sfc_temperature` and `surface_albedo`) so that we can easily understand and control them, without worrying about interdependencies and interactions.

Reply:
I don't think we can avoid a new surface model. Right now whether we apply surface flux or not is controlled by vertical diffusion, which is not ideal. So we need something else to specify it. This is different from `sfc_setup` as there are some cases where we want to have surface flux (for EDMF updraft calculations), but we don't want to apply the tendency. We may be able to incorporate it within `sfc_setup`, but that will make surface conditions more complicated. I will talk with Dennis to see what would be the easiest and not too confusing way for this.